Tests/Functional/SerializerErrorTest.php

```diff
@@ -87,6 +88,7 @@ public function serializeInvalidFormJsonProvider()
 
     /**
      * @dataProvider serializeInvalidFormXmlProvider
+     * @group legacy
```

Comment:
this should not be added in the legacy group. ``isBound`` was already deprecated in favor of ``isSubmitted`` in Symfony 2.3, so we should just update the code instead